src/mergesort.js

```diff
@@ -0,0 +1,33 @@
+export default function mergeSort(array) {
+  if (!Array.isArray(array)) return "We can only sort arrays"
+  
+  if(array.length < 2) return array
+
+  var middle = Math.floor(array.length / 2)
+  var left = array.slice(0, middle)
+  var right = array.slice(middle, array.length)
+
+  return stitch(mergeSort(left), mergeSort(right))
+}
+
+function stitch(left, right){
```

Comment:
I like how your implementation leverages the builtin `shift` function.  I encourage you to consider how you would implement this if no `shift` were available (which is typical of languages like Java and C++).